postgres-appliance/scripts/configure_spilo.py

```diff
@@ -107,10 +108,21 @@ def write_certificates(environment, overwrite):
             '-out',
             environment['SSL_CERTIFICATE_FILE'],
         ]
-        logging.info('Generating ssl certificate')
+        logging.info('Autogenerating ssl certificate')
         p = subprocess.Popen(openssl_cmd, shell=False, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
         output, _ = p.communicate()
         logging.debug(output)
+        logging.info('Generating ssl certificate')
+    if 'SSL_CA' in environment:
+        logging.info('Generating ssl ca certificate')
+        write_file(environment['SSL_CA'], environment['SSL_CA_FILE'], overwrite)
+    else:
+        logging.info('No ca certificate to generate')
+    if 'SSL_CRL' in environment:
+        logging.info('Generating ssl crl certificate')
+        write_file(environment['SSL_CRL'], environment['SSL_CRL_FILE'], overwrite)
+    else:
+        logging.info('No crl certificate to generate')
```

Comment:
This block should not be executed if is self-signed certificates are generated.
The better place for it would be here: https://github.com/zalando/spilo/pull/624/files#diff-e4dcf964214d58d733ae9874d4a2ab4b8e9eb99213307e1666af5dad21a326bbR92-R93

Also, default values of `SSL_CA_FILE` and `SSL_CRL_FILE` are empty strings.
If there is a content for them they should be set to `os.path.join(placeholders['RW_DIR'], 'certs', 'ca.crt'))` and `os.path.join(placeholders['RW_DIR'], 'certs', 'server.crl'))` respectively.